tests/rptest/scale_tests/many_topics_test.py

```diff
@@ -514,10 +592,7 @@ def _decommission_node_safely(self):
         self._standby_broker = node_to_decom
 
     def _get_partition_count(self):
```

Comment:
What does this function do? It's changing from a dynamic query of the partition count, to something which is static once _current_profile is set, right?